.github/workflows/build.yml

```diff
@@ -11,32 +11,81 @@ jobs:
+            cmake_args: >-
+              -DFAAD=ON
+              -DKEYFINDER=ON
+              -DLOCALECOMPARE=ON
+              -DMAD=ON
+              -DMODPLUG=ON
+              -DWAVPACK=ON
+            cmake_generator: Unix Makefiles
             ctest_args:
-            ccache_path: ~/.ccache
+            compiler_cache: ccache
+            compiler_cache_path: ~/.ccache
             cpack_generator: DEB
+            buildenv_basepath: /home/runner/buildenv
             buildenv_script: tools/ubuntu_buildenv.sh
             artifacts_name: Ubuntu 18.04 DEB
             artifacts_path: cmake_build/*.deb
+            qt_qpa_platform: offscreen
           - name: Ubuntu 20.04 (gcc)
             os: ubuntu-20.04
-            cmake_args: -DFFMPEG=ON -DKEYFINDER=ON -DLOCALECOMPARE=ON -DMAD=ON -DMODPLUG=ON -DWAVPACK=ON
+            cmake_args: >-
+              -DFFMPEG=ON
+              -DKEYFINDER=ON
+              -DLOCALECOMPARE=ON
+              -DMAD=ON
+              -DMODPLUG=ON
+              -DWAVPACK=ON
+            cmake_generator: Unix Makefiles
             ctest_args:
-            ccache_path: ~/.ccache
+            compiler_cache: ccache
+            compiler_cache_path: ~/.ccache
             cpack_generator: DEB
+            buildenv_basepath: /home/runner/buildenv
             buildenv_script: tools/ubuntu_buildenv.sh
             artifacts_name: Ubuntu 20.04 DEB
             artifacts_path: cmake_build/*.deb
+            qt_qpa_platform: offscreen
           - name: macOS 10.15
             os: macos-10.15
-            cmake_args: -DCOREAUDIO=ON -DHSS1394=ON -DMACOS_BUNDLE=ON
+            cmake_args: >-
+              -DCOREAUDIO=ON
+              -DHSS1394=ON
+              -DMACOS_BUNDLE=ON
+            cmake_generator: Unix Makefiles
             # TODO: Fix this broken test on macOS
             ctest_args: --exclude-regex DirectoryDAOTest.relocateDirectory
             cpack_generator: DragNDrop
-            ccache_path: /Users/runner/Library/Caches/ccache
+            compiler_cache: ccache
+            compiler_cache_path: /Users/runner/Library/Caches/ccache
+            buildenv_basepath: /Users/runner/buildenv
             buildenv_script: tools/macos_buildenv.sh
             artifacts_name: macOS DMG
             artifacts_path: cmake_build/*.dmg
+            qt_qpa_platform: offscreen
+          - name: Windows 2016 (MSVC)
+            os: windows-2016
+            cmake_args: >-
+              -DHSS1394=ON
+              -DKEYFINDER=OFF
+              -DLOCALECOMPARE=ON
+              -DMAD=ON
+              -DMEDIAFOUNDATION=ON
+              -DSTATIC_DEPS=ON
+            cc: cl
+            cxx: cl
+            cmake_generator: Ninja
+            # TODO: Fix these broken tests on Windows
+            ctest_args: --exclude-regex '^AutoDJProcessorTest.(FullIntroOutro_LongerOutro|EnabledSuccess_DecksStopped|FadeToDeck1_LoadOnDeck2_TrackLoadFailed|FadeToDeck2_LoadOnDeck1_TrackLoadSuccess)$'
```

Comment:
Have you investigated these test failures at all? Are they artifacts of the CI or testing setup or do they indicate real bugs? I'm fine merging this before having those questions answered, but I think we need to look into this before making a release.

Reply:
I had a superfluous look at them to make sure there is not obvious issue in the build workflow, nothing more.